src/coreclr/gc/unix/cgroup.cpp

```diff
@@ -59,30 +59,19 @@ class CGroup
 
     static char *s_memory_cgroup_path;
 
-    static const char *s_mem_stat_key_names[];
-    static size_t s_mem_stat_key_lengths[];
-    static size_t s_mem_stat_n_keys;
+    static char *s_mem_stat_inactive_file_key_name;
```

Comment:
Can you please keep this a `const char*`? Then you don't need the casts of string literals below to `char*`.